app/extras/app_config.rb

```diff
@@ -30,4 +30,17 @@ def self.theme
       accent_palette_config:  JSON.parse(ENV.fetch('THEME_ACCENT_PALETTE_CONFIG', '{"default": "500"}'))
     }
    end
+
+   def self.features
+     {
+       create_user:   env_boolean('FEATURES_CREATE_USER', true),
+       create_group:  env_boolean('FEATURES_CREATE_GROUP', true),
+       public_groups: env_boolean('FEATURES_PUBLIC_GROUPS', true),
+       help_link:     env_boolean('FEATURES_HELP_LINK', true)
+     }
+   end
+
+   def self.env_boolean(key, default)
+     %w(1 true).include? ENV.fetch(key, default).to_s
```

Comment:
For some reason %w(1 true) feels not the best to me; it's arbitrary and may or may not follow rules about ENV we've created already or will create.

```
# .env
FEATURES_CREATE_USER=1 # you can create users
# you can create groups because you didn't put anything
FEATURES_PUBLIC_GROUPS=yes # you can't see public groups
FEATURES_HELP_LINK=false # you can't see the help link
```

It also took me a bit to figure out that we're wanting to default to true here so we don't have to change existing installs at all. How would you feel about

```
def self.features
  {
    create_user: !ENV['DISABLE_FEATURE_CREATE_USER'],
    create_group: !ENV['DISABLE_FEATURE_CREATE_GROUP']
  }
end
```